features/bevragen/persoon/verblijfplaats/dev/fields-gba.feature

```diff
@@ -269,7 +267,7 @@ Rule: wanneer één of meerdere velden van een verblijfplaats buitenland wordt g
     | type                | RaadpleegMetBurgerservicenummer |
     | burgerservicenummer | 000000152                       |
     | fields              | <fields>                        |
-    Dan heeft de response een persoon zonder gegevens
+    Dan heeft de response een persoon zonder 'verblijfplaats' gegevens
```

Comment:
Hadden we vanochtend niet de afspraak gemaakt dat de GBA API in dit geval een lege verblijfplaats levert. Zo ja, dan dat hier corrigeren.